metaspace_converter/anndata_to_array.py

```diff
@@ -16,13 +17,15 @@ def _check_pixel_coordinates(adata: AnnData) -> bool:
     return np.all(np.equal(pixel_list, required_pixels))
 
 
-def anndata_to_image_array(adata: AnnData) -> np.ndarray:
+def anndata_to_image_array(adata: AnnData, layer: Optional[str]=None) -> np.ndarray:
```

Comment:
Also added `layer` argument to the `anndata_to_image_array` function. With this also the processed images from layers can be extracted.